recipe/meta.yaml

```diff
@@ -27,6 +27,7 @@ build:
 requirements:
   build:
     - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
```

Comment:
Side-note: The interest reader might have a look at https://github.com/conda-forge/conda-forge.github.io/issues/2102.